sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Statistics.scala

```diff
@@ -47,11 +47,13 @@ object Statistics {
  *                    defaults to the product of children's `sizeInBytes`.
  * @param rowCount Estimated number of rows.
  * @param attributeStats Statistics for Attributes.
+ * @param isAdaptive Statistics from AQE.
```

Comment:
"adaptive" sounds weird for statistics. How about
```
@param isRuntime Whether the statistics is inferred from query stage runtime statistics during
                 adaptive query execution.
```

Reply:
sounds better